### DISCUSSION TITLE
Alienfx loses sensor temperature parameters

### DISCUSSION THREAD

**Discussion by ZhangRuoyuan0311** [T-Troll/alienfx-tools#398] [Q&A]
Hello!

AW M18 will perform a bios check for fans, memory, etc. in random boots (the white screen). Every time after this bios check, my alienfx will lose all the other sensor temperature parameters that it was monitoring in fan settings, leaving only CPU, GPU and two drives I have: 
![Screenshot 2023-12-10 094634](https://github.com/T-Troll/alienfx-tools/assets/63335188/75cfb6f0-9ac7-40bd-871b-a56a7ab75fe1)

I have to reininstall Alienfx so that the other sensor parameters are back. Not sure if this is a bug or Dell rewrites some accesses from alienfx or why?

**Comment by T-Troll** [comment]
Re-install afx does nothing with config, so it's the same as simple restart it.
It looks like Intel DPTF doesn't start after you check, so these sensors not available from it. Not good at all, CPU power control lost as well...
And yes, it's a BIOS bug.

So i can recommend you just restart (or power off/on) system after check.
